app/dts/behaviors/ug_indicators.dtsi

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/hid_indicators.h>
+
+/ {
```

Comment:
Great work on this PR, I use it for every build :)

I would love it if there was also a behavior that makes so color is changed for the specified key if ctrl, shift, or alt is pressed 

would also be great if I could make a key change color if the key itself if pressed